### PR TITLE
Update Get-NetTCPSetting.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue - "fill in the blank" assignments - Description has been updated now.

### CODE REVIEW

**Comment by get-itips** [comment]
Agree with @dariomws306

File: docset/winserver2012r2-ps/nettcpip/Get-NetTCPSetting.md [hunk 1]
```diff
@@ -107,7 +107,9 @@ Accept wildcard characters: False
 ```
 
 ### -AutoReusePortRangeNumberOfPorts
-{{Fill AutoReusePortRangeNumberOfPorts Description}}
+Specifies the number of ports for the auto-reuse port range, which is a port range used for local ephemeral port selection by outbound TCP connections for which either SO_REUSE_UNICASTPORT has been set on the socket, or for which connect() has been called without calling bind() on the socket.
+
+If you specify 0, the auto-reuse feature is disabled and ephemeral ports are drawn instead from the dynamic port range as specified by DynamicPortRangeStartPort and DynamicPortRangeNumberOfPorts, even if SO_REUSE_UNICASTPORT is set on a socket.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/nettcpip/Get-NetTCPSetting.md]:
```suggestion
If you specify 0, the auto-reuse feature is disabled and ephemeral ports are drawn instead from the dynamic port range as specified by _DynamicPortRangeStartPort_ and _DynamicPortRangeNumberOfPorts_, even if SO_REUSE_UNICASTPORT is set on a socket.
```

**Comment by tseward** [comment]
Also agree with Agree with @dariomws306

**Comment by sowjanya365** [comment]
Hi @dariomws306 @tseward 

I've updated the document with the suggested modifications. Kindly review. Thank you.

**Comment by dariomws306** [comment]
Looks good now.

Good work!

**Comment by rjagiewich** [comment]
Hi @e0i @msbemba, per the same comment we made in #1343, we have been merging with master before pushing to live. Can we get this into master first? It looks fine to merge otherwise. 
cc @dstrome
thanks! -Rebecca

**Comment by dstrome** [comment]
@e0i @msbemba - We only accept PRs to the master branch so this PR is being closed without merging. Please submit a new PR against the master branch. Thanks.

**Pull request closed without merging**